cmd/buildah/config.go

```diff
@@ -63,6 +63,10 @@ var (
 			Name:  "shell",
 			Usage: "add `shell` to run in containers",
 		},
+		cli.StringFlag{
+			Name:  "stop-signal",
+			Usage: "add `signal` to stop containers",
```

Comment:
set default stop signal for container.  This signal will be sent when container is stopped, default is SIGINT.